frontend/src/libs/components/pagination/pagination.tsx

```diff
@@ -57,9 +60,9 @@ const Pagination: React.FC<Properties> = ({
   );
 
   const handleChangePageSize = useCallback(
-    (value: string | undefined) => {
-      if (value && !Number.isNaN(+value) && onChangePageSize) {
-        onChangePageSize(+value);
+    (option: SingleValue<SelectOption>) => {
+      if (option?.value && !Number.isNaN(+option.value) && onChangePageSize) {
```

Comment:
do you need check if option is optional? in params you described it as defined value

Reply:
there is lint error without null check